exercise.wwwapi/Endpoints/ProductEndpoints.cs

```diff
@@ -0,0 +1,105 @@
+﻿using exercise.wwwapi.Models;
+using exercise.wwwapi.Repository;
+using Microsoft.AspNetCore.Mvc;
+using System.Runtime.CompilerServices;
+
+namespace exercise.wwwapi.Endpoints
+{
+    public static class ProductEndpoints
+    {
+        public static void ConfigureProductEndpoints( this WebApplication app)
+        {
+            var productGroup = app.MapGroup("Products");
+
+            productGroup.MapGet("/products", GetProducts);
```

Comment:
no need to add `products` in here. As with this phrase the URL to make the calls looks like this. 
GET All
`https://localhost:1234/product/products`
GET / DELETE / PUT
`https://localhost:1234/product/products/1`
POST 
`https://localhost:1234/product/products

Reply:
fixed